src/Striot/CompileIoT.hs

```diff
@@ -164,7 +164,8 @@ generateSinkFn:: StreamGraph -> String
 generateSinkFn sg = "sink1 :: Show a => Stream a -> IO ()\nsink1 = " ++
     (intercalate "\n" $ parameters $ head $ reverse $ vertexList sg) ++ "\n"
 
-generateNodeLink n = "main = nodeLink streamGraphFn 9001 \"node"++(show n)++"\" 9001"
+generateNodeLink :: Integer -> String
```

Comment:
Int probably sufficient (to match generateNodeSink) but not important

Reply:
I initially tried this, but they are actually different types. The `generateNodeSink` function takes `valence :: Int` generated from `partValence :: StreamGraph -> [StreamGraph] -> Int` but `generateNodeLink` and `generateNodeSrc` take `partId :: Integer` from the definition of `generateCodeFromStreamGraph ...`.

It's probably relatively straightforward to convert them all to the same type but I didn't want to make too many adjustments to this.